src/browser/FileTree.js

```diff
@@ -32,6 +31,32 @@ class FileTree extends LitElement {
           position: relative;
         }
 
+        .codicon[class*="codicon-"] {
```

Comment:
hardcode the codicon CSS parts that we need instead of the entire sheet, saves some kb